contrib/gcs_importer.py

```diff
@@ -77,7 +77,7 @@ def setup_sketch(timeline_name, index_name, username, sketch_id=None):
         sketch_id: (str) Optional sketch_id to add timeline to
 
     Returns:
-        (str) Sketch ID
+        (str) Sketch ID and Timeline ID
```

Comment:
technically it doesn't return a string, but a tuple with the sketch ID and the timeline ID (aren't both of them integers?)